anchore_engine/services/apiext/api/controllers/images.py

```diff
@@ -1207,6 +1219,40 @@ def analyze_image(account, source, force=False, enable_subscriptions=None, annot
         raise err
 
 
+def images_imageDigests(request_inputs, imageDigests):
```

Comment:
Can we just use a better model here? I'd prefer to not continue the non-verb specific function decomposition in this file, and move to a format that is actually testable in unit/integration tests.

Reply:
I went between making this way more simpler/cleaner vs fitting into the current model. I even have the changes in a different branch as I didn't want to lose them entirely. Will switch it